### ISSUE TITLE
Access is denied or unhandled exception

### ISSUE THREAD

**Issue by Grimington** [ChrisTitusTech/winutil#775]
**Describe the bug**
Is it normal that during the process of tweaking/disabling telemetry and so on the program in the background runs such lines in yellow? I running a clean install of windows 11. granted amin rights on CTT-Windows Utility.exe
After tweaking my processes are 110-120ish. 

Example: 

WARNING: Unable to set SgrmBroker due to unhandled exception
WARNING: The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
Setting Services SharedAccess to Manual
Setting Services SharedRealitySvc to
WARNING: Unable to set SharedRealitySvc due to unhandled exception
WARNING: The argument is null or empty. Provide an argument that is not null or empty, and then try the command again.
Setting Services ShellHWDetection to Automatic
Setting Services SmsRouter to Manual
Setting Services Spooler to Automatic
Setting Services SstpSvc to Manual
Setting Services StateRepository to Manual
WARNING: Unable to set StateRepository due to unhandled exception
WARNING: Service 'State Repository Service (StateRepository)' cannot be configured due to the following error: Access
is denied

**To Reproduce**
select Laptop tweaks and run

**Comment by DeveloperDurp** [comment]
Currently Chris is working on adding a lot of additional values to be able to reset windows back to default when running the undo tweaks so you will see some messages stating "the argument is null or empty." You can safely ignore these.

For the access denied I am not familiar enough with the StateRepository service, it's possible windows prevented you from using changing it because it was in use or something like that. You can either try to set it manually to see or ignore the warning.

**Comment by Grimignton** [comment]
Thank you. DeveloperDurp,
I've remembered that on windows 10 (few years before) wasn't present and assume that I have done something (before this tool was considered a virus via windows and it was a slight pain to be sure that you actually give it proper environment to work)
I've pasted just a small part of this "example" as the list is large and its full of this yellow Warnings. And I think its way above my head to try to tweak on my own. Plus i'm currently trying windows 11 to check if its better than 10 in games :D. But as long as its working - i trust Chris and Panjo
